bblfsh/client.py

```diff
@@ -1,54 +1,57 @@
 import os
-import sys
+import typing as t
 
 import grpc
 
-from bblfsh.aliases import (ParseRequest, ParseResponse, NativeParseRequest, NativeParseResponse,
-        VersionRequest, ProtocolServiceStub, SupportedLanguagesRequest, SupportedLanguagesResponse)
-from bblfsh.sdkversion import VERSION
-
-# The following two insertions fix the broken pb import paths
-sys.path.insert(0, os.path.join(os.path.dirname(__file__),
-                                "gopkg/in/bblfsh/sdk/%s/protocol" % VERSION))
-sys.path.insert(0, os.path.dirname(__file__))
+from bblfsh.aliases import (ParseRequest, DriverStub, ProtocolServiceStub,
+                            VersionRequest, SupportedLanguagesRequest, ModeType,
+                            VersionResponse)
+from bblfsh.result_context import ResultContext
 
 
 class NonUTF8ContentException(Exception):
     pass
 
 
-class BblfshClient(object):
+class BblfshClient:
     """
-    Babelfish gRPC client. Currently it is only capable of fetching UASTs.
+    Babelfish gRPC client.
     """
 
-    def __init__(self, endpoint: str):
+    def __init__(self, endpoint:str) -> None:
         """
         Initializes a new instance of BblfshClient.
 
         :param endpoint: The address of the Babelfish server, \
                          for example "0.0.0.0:9432"
+        :type endpoint: str
         """
+
         self._channel = grpc.insecure_channel(endpoint)
-        self._stub = ProtocolServiceStub(self._channel)
+        self._stub_v1 = ProtocolServiceStub(self._channel)
+        self._stub_v2 = DriverStub(self._channel)
```

Comment:
This is unrelated to the PR but maybe you can add at once:

```python
def close(self):
    """
    Close the gRPC channel and free the acquired resources. Using a closed client is not supported.
    """
    self._channel.close()
    self._channel = self._stub_v1 = self._stub_v2 = None
```

We need this function in many different places and currently call `_channel.close()`.

Reply:
Done